integrations/operator/crdgen/testdata/protofiles/teleport/legacy/types/types.proto

```diff
@@ -3081,14 +3081,8 @@ message LoginStatus {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "lock_expires,omitempty"
   ];
-  // RecoveryAttemptLockExpires contains the time when this lock will expire
-  // from reaching MaxAccountRecoveryAttempts. This field is used to determine
-  // if a user got locked from recovery attempts.
-  google.protobuf.Timestamp RecoveryAttemptLockExpires = 5 [
-    (gogoproto.stdtime) = true,
-    (gogoproto.nullable) = false,
-    (gogoproto.jsontag) = "recovery_attempt_lock_expires,omitempty"
-  ];
+  reserved 5;  // removed RecoveryAttemptLockExpires after lockout was removed
```

Comment:
Usually I see these reserve blocks at the top of the message - but unsure if that's a standard thing.

Reply:
AFAIK this goes either way. Not sure if we have a consensus.